src/component/container/app.tsx

```diff
@@ -145,54 +140,51 @@ export class App extends React.Component {
 		const DevTools = this.getDevTools();
 
 		return (
-			<Layout directionVertical handleClick={this.handleMainWindowClick}>
+			<Layout directionVertical onClick={this.handleMainWindowClick}>
 				<ChromeContainer />
-
 				<MainArea>
-					{project && [
-						<SideBar key="left" directionVertical hasPaddings>
-							<ElementPane>
-								<ElementList />
-							</ElementPane>
-							<PatternsPane>
-								<PatternListContainer />
-							</PatternsPane>
-						</SideBar>,
-						<PreviewPaneWrapper key="center" previewFrame={previewFramePath} />,
-						<SideBar key="right" directionVertical hasPaddings>
-							<PropertyPane>
-								<PropertyList />
-							</PropertyPane>
-						</SideBar>
-					]}
-
-					{!project && (
-						<SplashScreen>
-							<Space sizeBottom={SpaceSize.L}>
-								<Headline textColor={colors.grey20} order={2}>
-									Getting started with Alva
-								</Headline>
-							</Space>
-							<Space sizeBottom={SpaceSize.XXXL}>
-								<Copy size={CopySize.M} textColor={colors.grey20}>
-									You can open an existing Alva space or create a new one based on our
-									designkit including some basic components to kickstart your project.
-								</Copy>
-							</Space>
-							<Space sizeBottom={SpaceSize.S}>
-								<Button handleClick={this.handleCreateNewSpaceClick} order={Order.Primary}>
-									Create new Alva space
-								</Button>
-							</Space>
-							<Link color={colors.grey50} onClick={this.handleOpenSpaceClick}>
-								<Copy size={CopySize.S}>or open existing Alva space</Copy>
-							</Link>
-						</SplashScreen>
+					{project ? (
+						<React.Fragment>
+							<SideBar
+								key="left"
+								directionVertical
+								onClick={() => store.setSelectedElement()}
+							>
+								<ElementPane>
+									<ElementList />
+									<AddButton
```

Comment:
I thought an active state would be cooler, to not disturb the user because the button is sometimes not there. But let’s test that as well (see below)!